packit/utils/extensions.py

```diff
@@ -0,0 +1,74 @@
+# MIT License
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import json
+from typing import Any
+
+from packit.exceptions import PackitException
+
+
+def get_rev_list_kwargs(opt_list):
```

Comment:
Where are `get_rev_list_kwargs` and `nested_get` used? Cannot we move those two near the code they are used?

Reply:
Some comment above, they're not used at all (**in `packit`**). However since `packit-service` requires `set_logging`, I'd rather look into that before merging if there is no other reference there.

Reply:
-  `get_rev_list_kwargs` - no reference in `packit-service` through git grep
-  `nested_get` - found it in `packit-service/worker/parser.py`

Reply:
Then I vote for removing `get_rev_list_kwargs`